SgmlReaderDll/SgmlReader.cs

```diff
@@ -1048,14 +1089,38 @@ public override string XmlLang
         }
 
         /// <summary>
-        /// Specifies how white space is handled.
+        /// Specifies how whitespace nodes are handled.
         /// </summary>
         public WhitespaceHandling WhitespaceHandling
         {
             get => _whitespaceHandling;
             set => _whitespaceHandling = value;
         }
 
+        private TextWhitespaceHandling _textWhitespace;
+
+        /// <summary>
+        /// Specifies how leading and trailing whitespace in <c>InnerText</c> and <c>Value</c> properties is handled.
+        /// </summary>
+        /// <remarks>This property is intended to make it easier to process files such as OFX feeds with self-closing elements separated by line-breaks as it allows <c>InnerText</c> to be passed directly into methods that expect string inputs to be trimmed of whitespace as a precondition.</remarks>
```

Comment:
I like to line wrap at about 120, makes these kinds of code reviews easier.

Reply:
My primary monitor is [Dell's 43-inch 4K behemoth](https://www.amazon.com/Dell-U4320Q-UltraSharp-Business-Warranty/dp/B084LB7RN2) that I run at 96dpi so I never have to do horizontal scrolling :)

But sure!

Reply:
Yeah, sorry, my eyes are too old to have so much text on one screen... :-)